.github/workflows/kata-deploy.yaml

```diff
@@ -0,0 +1,34 @@
+on: push
+
+name: Build, Test, and Publish kata-deploy
+
+jobs:
+  kata-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: build-and-push-kata-deploy-ci
+        run: |
+          git clone https://github.com/amshinde/kata-packaging
+          pushd ./kata-packaging
+          git checkout 1.9.0-rc0
```

Comment:
Hard-coded version number. Can this file be generated to avoid this?

Reply:
this file isn't intended to be merged. It's there as an FYI (see the commits themselves)